index.js

```diff
@@ -327,7 +327,7 @@ function formatAnswer(reportData, url, scan, options) {
     scan.score = newScore;
   }
 
-  if (options.nagios != undefined) {
+  if (options.nagios !== undefined) {
```

Comment:
You may want to test this. I can never remember the exact rules for `undefined` and `===` comparisons. Or maybe we can just do `if (options.nagios) { ... }`.